docs/k8s-docker-for-mac.md

```diff
@@ -78,7 +78,8 @@ by adding the stanza below to mycluster.yaml:
 ```yaml
 k8s:
   persistence:
-    defaultStorageClass: hostpath
+    hasDefaultStorageClass: false
+    explicitStorageClass: hostpath
```

Comment:
With your PR, this stanza is not required even for docker-for-mac 18.06. I think we can delete it.